### PR TITLE
RHDEVDOCS-4703-edit-prometheusscrapebodysizelimithit-runbook

### PR DESCRIPTION
**Pull request by bburt-rh**
Language and style edits for this runbook.

### CODE REVIEW

File: alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md [hunk 1]
```diff
@@ -2,39 +2,41 @@
...
 
-The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of
-Prometheus' scrape targets replies with a response body larger than the
-configured `body_size_limit`.
-By default, there is no limit on the body size of the scraped targets.
-When set, this limit prevents Prometheus from consuming excessive amounts of memory
-when scraped targets return a response that is deemed too large.
+The `PrometheusScrapeBodySizeLimitHit` alert triggers when at least one
+Prometheus scrape target replies with a response body larger than the
+value configured in the `body_size_limit` field in the
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md:7]:
```suggestion
value configured in the `enforcedBodySizeLimit` field in the
```

File: alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md [hunk 2]
```diff
@@ -2,39 +2,41 @@
...
+`cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
+
+By default, no limit exists on the body size of scraped targets. When a value
+is defined for `body_size_limit`, this limit prevents Prometheus from consuming
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md:11]:
```suggestion
is defined for `enforcedBodySizeLimit`, this limit prevents Prometheus from consuming
```

File: alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md [hunk 3]
```diff
@@ -78,74 +80,80 @@ shown in the example below:
...
+    ```
 
 ## Mitigation
 
-According to the result of the investigation, the cause can be either the limit is
-too small or there is a bug in the target causing it to report too many metrics.
+Your analysis of the issue might reveal that the alert was triggered by one of
+two causes:
+
+* The value set for `body_size_limit` is too small.
```
Comment (simonpasquier) [inline alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md:145]:
```suggestion
* The value set for `enforcedBodySizeLimit` is too small.
```

Comment (bburt-rh) [reply alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md]: fixed

File: alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md (hunk 2 above)
Comment (bburt-rh) [reply alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md]: fixed

**Pull request merged**